core/src/test/scala/unit/kafka/server/ApiVersionTest.scala

```diff
@@ -0,0 +1,51 @@
+/**
+  * Licensed to the Apache Software Foundation (ASF) under one or more
+  * contributor license agreements.  See the NOTICE file distributed with
+  * this work for additional information regarding copyright ownership.
+  * The ASF licenses this file to You under the Apache License, Version 2.0
+  * (the "License"); you may not use this file except in compliance with
+  * the License.  You may obtain a copy of the License at
+  *
+  *    http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+
+package unit.kafka.server
+
+import kafka.server.KafkaApis
+import org.apache.kafka.common.requests.ApiVersionResponse.ApiVersion
+import org.apache.kafka.common.protocol.{Protocol, ApiKeys}
+import org.junit.Test
+
+class ApiVersionTest {
```

Comment:
Sure, that makes sense. I have tested this with java client changes, which are part of #1251, and Magnus has tested it with librdkafka. However, I am not sure if Magnus has tested with the latest version or not. I do not mean to say that with those tests, unit tests are not required. Would you guys be ok if I create a follow up JIRA to add the tests, so that this PR does not get blocked? I will provide PR with tests as soon as #1095 gets in. This PR is a blocker for #1251.